### PR TITLE
Speed up debug attach by setting outFiles

### PR DESCRIPTION
**Pull request by jakebailey**
This improves the startup time of the debugger. This is a template, so everyone will have to manually copy this if they want it.

(I had this locally after it was suggested by @connor4312 and then proceeded to entirely forget to send this.)

### CODE REVIEW

File: .vscode/launch.template.json [hunk 1]
```diff
@@ -40,6 +40,10 @@
       "env": {
         "NODE_ENV": "testing"
       },
+      "outFiles": [
+        "${workspaceFolder}/built/**/*.js",
```
Comment (jakebailey) [inline .vscode/launch.template.json:44]: For future proofing maybe `{js,mjs,cjs}` though...

Comment (DanielRosenwasser) [reply .vscode/launch.template.json:44]:
```suggestion
        "${workspaceFolder}/built/**/*.{js,mjs,cjs}",
```

**Review by DanielRosenwasser — APPROVED** [review]

**Comment by connor4312** [comment]
Lgtm

File: .vscode/launch.template.json (hunk 1 above)
Comment (jakebailey) [reply .vscode/launch.template.json]: Oops, this syntax doesn't actually work in this position. I guess these globs are different?

**Pull request merged**